src/commands/Checkin.ts

```diff
@@ -85,10 +86,6 @@ export default class Checkin extends Command {
       //
       // We need two sets of arrays for "checkin":
```

Comment:
Can update this comment and related ones: we don't need two sets of arrays for "checkin" anymore